src/cobertura.rs

```diff
@@ -41,6 +42,20 @@ struct CoverageStats {
     complexity: f64,
 }
 
+impl std::ops::Add for CoverageStats {
```

Comment:
This could be derived using https://crates.io/crates/derive_more. Just let me know if you would prefer that.